docs/imagej_python.rst

```diff
@@ -11,32 +11,125 @@ This is much easier that when using the Java API and Bio-Formats plugin.
 
 We will show in the notebooks:
 
-- How to start ImageJ in Python
+- How to start ImageJ in Python.
 
-- How to load data from OMERO
+- How to load data from OMERO.
 
-- How to run ImageJ macro from Python
+- How to run ImageJ macro from Python.
 
 Setup
 -----
 
-In this case, Fiji has been installed locally. It is possible to install Fiji "on the fly",
-see `ImageJ Tutorials <https://nbviewer.jupyter.org/github/imagej/tutorials/blob/master/notebooks/ImageJ-Tutorials-and-Demo.ipynb>`_ for more options.
+For using the Python examples and notebooks of this guide we recommend using Conda (Option 1).
+Conda manages programming environments in a manner similar to 
+`virtualenv <https://virtualenv.pypa.io/en/stable/>`_.
+
+Alternatively you can use ``repo2docker`` to build and run a Docker image locally (Option 2).
+This Docker image will provide the Conda environment and Jupyter notebooks with some image 
+analysis workflows.
+
+*Option 1*
+~~~~~~~~~~
 
 Install omero-py and `pyimagej <https://pypi.org/project/pyimagej/>`_ via Conda:
 
 - Install `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ if necessary.
 
 - Create a programming environment using Conda and activate it::
 
-    $ conda create -n fiji_python python=3.6
+    $ conda create -n imagej_python python=3.6
 
-    $ conda activate fiji_python
+    $ conda activate imagej_python
 
 - Install omero-py and `pyimagej <https://pypi.org/project/pyimagej/>`_::
 
     $ conda install -c conda-forge pyimagej
 
     $ conda install -c ome omero-py
 
+*Option 2*
+~~~~~~~~~~
+
+Create a local Docker Image using ``repo2docker``, see :download:`README.md <https://github.com/ome/omero-guide-python/blob/master/README.md>`::
```

Comment:
In the README, which is hosted in omero-guides-python is a sentence "clone this repository". But in the sense of this workflow, the omero-guides-python repo does not need to be cloned (note that when you click onto the README, it does not download the file locally, instead, opens it in github)